bootloader.mk

```
@@ -90,7 +90,7 @@ ifeq ($(strip $(BOOTLOADER)), USBasp)
 endif
 ifeq ($(strip $(BOOTLOADER)), lufa-ms)
     OPT_DEFS += -DBOOTLOADER_MS
-    BOOTLOADER_SIZE = 6144
+    BOOTLOADER_SIZE = 8192
```

I'm opposed to changing this outright, because this can cause issues for older boards (such as the tada68).

Instead, I'd say that this should be configurable, and set in the dztech boards.
```suggestion
    BOOTLOADER_SIZE ?= 6144
```

---

At least until LUFA is also updated.

---

aight

---

IMO it should be the other way around - the existing boards should have it set to 6144.